### ISSUE TITLE
Scip version differences

### ISSUE THREAD

**Issue by Missthepast** [scipopt/scip#105]
May I ask if there is a significant difference between version 7.0.0 and the latest version? I have a program that calls a different version of Scip, resulting in a significant difference. How can I choose a more suitable version to use? Looking forward to your reply. Thank you

this is 7.0.0
![image](https://github.com/user-attachments/assets/280d3b95-ebc7-45a8-bb3a-f21d83b8a4c7)
this is 8.0.0
![image](https://github.com/user-attachments/assets/d391f3a7-efd5-4051-8631-bc61998e2a44)

**Comment by svigerske** [comment]
Yes, there are significant differences. See https://optimization-online.org/2021/12/8728/ and https://www.scipopt.org/doc-8.1.0/html/RN80.php for a description about the changes.

From what you post, I would say you should use the latest version. So many more zero. Sparsity is great.

**Comment by Missthepast** [comment]
![image](https://github.com/user-attachments/assets/157ca804-c4f2-41e6-9601-9c876bafc2b7)
Thank you very much for your reply. Do I have another question to ask here?
I have installed SCIP 7.0.0 on both Ubuntu containers using SCIPOptSuite-7.0.0-Lux.deb installation method, but I am using a script
When the solver=pe. SolverFactory ('scip ') calls Scip, one of them encounters a problem. Why can't it be found? However, I have already installed it in the environment. Do I need any other operations? I'm very sorry to disturb you
![image](https://github.com/user-attachments/assets/f32bf017-f455-4fed-bcbd-7f40f5762131)
